resources/js/views/admin/BoundingBox.vue

```diff
@@ -11,35 +11,30 @@
                     <div id="image-wrapper"
                          :style="image"
                          @mousedown.self="startDrawingBox"
-                         @mousemove="drawBox"
-                         @mouseup="stopDrawingBox"
+                         @mousemove="mouseMove"
+                         @mouseup="mouseUp"
                     >
 
                         <Box
                             v-if="drawingBox.active"
-                            :top="drawingBox.top"
-                            :left="drawingBox.left"
-                            :width="drawingBox.width"
-                            :height="drawingBox.height"
+                            :geom="drawingBox.geom"
                         />
 
                         <Box
                             v-for="(box, i) in boxes"
                             :key="i + box.height"
+                            :geom="box.geom"
                             :index="i"
-                            :top="box.top"
-                            :left="box.left"
-                            :width="box.width"
-                            :height="box.height"
                             :selected="box.selected"
                             :activeTop="box.activeTop"
                             :activeLeft="box.activeLeft"
                             :activeBottom="box.activeBottom"
                             :activeRight="box.activeRight"
-                            @select="selectBox(i)"
+                            @select="selectBox"
                             @activate="activate"
                             @deselectNode="deselectNode"
                             @repositionTop="repositionTop"
+			    @dragEnd="dragEnd"
```

Comment:
Where did `dragEnd` define?

Reply:
It's defined in the methods section, line 248.

The box child component emits a 'dragEnd' event when the user lifts the mouse button